src/dmd/dtemplate.d

```diff
@@ -476,6 +476,19 @@ extern (C++) final class Tuple : RootObject
 {
     Objects objects;
 
+    extern (D) this() {}
+
+    /**
+     *  Constructs a new instance.
```

Comment:
This line is spurious, as that's what a `this` constructor does. Please delete it.

Reply:
ok done.

Reply:
I think all documented symbols should have a summary. This allows to automatically generate an index table, i.e. something like the table at the top of [1].

[1] https://dlang.org/phobos/std_array.html